packages/protocol/contracts/layer1/preconf/impl/PreconfSlasher.sol

```diff
@@ -258,7 +257,7 @@ contract PreconfSlasher is IPreconfSlasher, EssentialContract {
 
         ITaikoInbox.Batch memory nextBatch = taikoInbox.v4GetBatch(uint64(_payload.batchId + 1));
         require(
-            keccak256(abi.encode(nextBatch.metaHash)) == evidence.nextBatchMetadata.infoHash,
+            keccak256(abi.encode(evidence.nextBatchMetadata)) == nextBatch.metaHash,
```

Comment:
why `vidence.nextBatchMetadata.proposedAt > _payload.preconferSlotTimestamp,` matters in line 168?

Reply:
I see. Previously I was assuming when a preconfer proposes two consecutive batches, the last block of each batch must be tagged with EOP. I see now.